src/awkward/_behavior.py

```diff
@@ -100,29 +100,32 @@ def find_ufunc_generic(
 
 
 def find_ufunc(behavior: Mapping | None, signature: tuple) -> UfuncLike | None:
-    if all(s is not None for s in signature):
-        behavior = overlay_behavior(behavior)
+    if any(s is None for s in signature):
+        return None
```

Comment:
If a signature contains any `None` value, give up on it.

I don't remember a reason for a signature to have a `None` value. Does this ever happen? Why? What does it mean?

Reply:
`None` is the signature type given to an Awkward `Content` that has no names and is not a `NumpyArray`.